week13/src/Java8Main.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Week 13 - Java8
+ *
+ * This program reads a file chosen by the user containing people's names and date of birth.
+ * The data is sorted alphabetically and filtered by a certain month.
+ * The final list is printed in a file chosen by the user.
+ *
+ * @author Florin
+ */
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+public class Java8Main {
+    public static void main(String[] args) throws IOException {
+
+        String []argument  = userInputs();
+        String inFile = argument[0];
+        String month = argument[1];
+        String outFile = argument[2];
+
+        List <Person> people = readFromFile(inFile);
+        List <Person> refinedList = people.stream().
+                sorted(Comparator.comparing(p -> p.getFirstName())).
```

Comment:
Excellent work with streams. I think (not sure) it's better to sort after you filter.